C Programming/Lab 1/InOut.h

```diff
@@ -1,23 +1,22 @@
 #include "stdio.h"
 #include "stdlib.h"
 
-int read_from_file(float ArrayOfNumbers[])
+void read_from_file(float ArrayOfNumbers[], int *NumberOfElements)
```

Comment:
This is about Pointers and other mess from Telegram chat. Skip it if you don't remember